experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -47,10 +48,7 @@ class ExPlat internal constructor(
      * If the provided [Experiment] was not included in [experiments], then [Control] is returned.
      * If [isDebug] is `true`, an [IllegalArgumentException] is thrown instead.
      */
-    fun getVariation(
-        experiment: Experiment,
-        shouldRefreshIfStale: Boolean = false,
```

Comment:
Think (🤔): Interestanto, I'll wait for how the APIs ends-up being in the end, also, we might want to update the KDoc for this function.